cmd/cmd.go

```diff
@@ -95,7 +96,8 @@ func Init(cmd *cobra.Command) (err error) {
 		if len(conf.File.Filename) != 0 {
 			atomic.StoreUint64(&hasLogFile, 1)
 			summary.InitCollector(true)
-			cmd.Printf("Detail BR log in %s\n", conf.File.Filename)
+			// cmd.PrintErr prints to stderr, but PrintErrf prints to stdout.
+			cmd.PrintErr(fmt.Sprintf("Detail BR log in %s \n", conf.File.Filename))
```

Comment:
lol a bug fix spf13/cobra#894 has been submitted but stay unmerged for over a year because some dependencies now relied on the bug (classic [xkcd-1172](https://xkcd.com/1172/) scenario)

Reply:
A bug becomes a feature 🤯